src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/NewUser.vue

```diff
@@ -6,22 +6,15 @@
           <h3>Profile Information</h3>
           <p class="small">The Registry will notify you of important updates regarding your certification</p>
         </div>
-        <v-form ref="form" validate-on="input">
+        <v-form ref="form" v-model="isValid" validate-on="input">
           <div class="d-flex flex-column ga-2">
-            <v-text-field
-              v-model="email"
-              label="Email"
-              variant="outlined"
-              color="primary"
-              type="email"
-              :rules="[Rules.email(), Rules.required()]"
-            ></v-text-field>
+            <v-text-field v-model="email" label="Email" variant="outlined" color="primary" type="email" :rules="emailRules"></v-text-field>
             <v-text-field
               v-model="phoneNumber"
               label="Phone Number"
               variant="outlined"
               color="primary"
-              :rules="[Rules.phoneNumber(), Rules.required()]"
```

Comment:
I think we could just do 
Rules.phoneNumber("Enter your primary 10 digit phone number") directly into the method here. 

Then we don't need to create the computed + method customPhoneRules + customEmailRules.